packages/sdk-themes/monite-theme.d.ts

```diff
@@ -0,0 +1,26 @@
+import * as React from 'react';
+
+import '@mui/material/styles';
+import '@mui/x-data-grid/themeAugmentation';
+
+declare module '@mui/material/styles' {
+  interface TypographyVariants {
+    label1: React.CSSProperties;
+    label2: React.CSSProperties;
+    label3: React.CSSProperties;
```

Comment:
We must not add extra variants.
Our customers will be obliged to use such `augmentation` somehow in their applications, and they will certainly use our types for their MUI components as well.

I understand that your decision was dictated by design, but the discrepancies between design and architecture need to be addressed at the design stage - we are completely tied to the MUI types and had no plans to change anything in favor of design.

Reply:
To summarize: at the meeting, we came to the understanding that additional options could be added to simplify future styling when new themes are added, but we only expand the options to those that are used.
Thanks for clarifying the path in making your decision.